<launch>
    <!-- Launch rviz waypoint visualizer node -->
    <node pkg="ros_car_control" type="waypoint_visualizer.py" name="waypoint_visualizer" output="screen"/>
    <node pkg="rviz" type="rviz" name="rviz" output="screen"/>
</launch>
